src-java/floodlight-service/floodlight-modules/README.md

```diff
@@ -15,3 +15,45 @@ where:
  ratePkts - is the rate in packets for specific type of rule
  disco_packet_size - the size of discovery packet, is being defined in properties file with the key 'org.openkilda.floodlight.switchmanager.SwitchManager.disco-packet-size'.
 ```
+
+#### How to debug Floodlight component
+
+In order to run floodLight in the debug mode we have to add this line of arguments
+```
+"-agentlib:jdwp=transport=dt_socket,address=50506,suspend=n,server=y”
+``` 
+when running the app.
```

Comment:
Here and after line 27, I would recommend not to split sentences with code blocks. There is an inline block with singular ` for inline code. If you need a code block for easier copy-pasting, then consider rewriting the sentence in way, so that the block comes after the thought is completed. In this case: ... add the following arguments when running the app: code block here. In the next case, I suggest to use an inline block.

Reply:
thanks, fixed

Reply:
The problem with splitting the sentence is still there. (that's because the arguments line is too long and doesn't have spaces)

Reply:
fixed, thank you